### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Spuddyminer** [tellor-io/telliot-feed-examples#182]
Log (running on mumbai)
```
Traceback (most recent call last):
  File "/home/ubuntu/tenv_dev/bin/telliot-examples", line 8, in <module>
    sys.exit(cli())
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_core/cli/utils.py", line 18, in wrapper
    return asyncio.run(f(*args, **kwargs))
  File "/usr/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_feed_examples/cli.py", line 414, in report
    _, _ = await reporter.report()
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_feed_examples/reporters/interval.py", line 470, in report
    _, _ = await self.report_once()
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_feed_examples/reporters/interval.py", line 351, in report_once
    await datafeed.source.fetch_new_datapoint()
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_feed_examples/sources/price_aggregator.py", line 83, in fetch_new_datapoint
    datapoints = await self.update_sources()
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_feed_examples/sources/price_aggregator.py", line 69, in update_sources
    inputs = await gather_inputs()
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_feed_examples/sources/price_aggregator.py", line 64, in gather_inputs
    datapoints = await asyncio.gather(
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_core/pricing/price_source.py", line 32, in fetch_new_datapoint
    datapoint = await self.service.get_price(self.asset, self.currency)
  File "/home/ubuntu/tenv_dev/lib/python3.9/site-packages/telliot_feed_examples/sources/price/spot/gemini.py", line 60, in get_price
    r = GeminiPriceResponse.parse_obj(d["response"])
  File "pydantic/main.py", line 511, in pydantic.main.BaseModel.parse_obj
  File "pydantic/main.py", line 331, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 4 validation errors for GeminiPriceResponse
bid
  field required (type=value_error.missing)
ask
  field required (type=value_error.missing)
last
  field required (type=value_error.missing)
volume
  field required (type=value_error.missing)
```

**Comment by tallywiesenberg** [comment]
@Spuddyminer 
- what did you run to get this bug?
- which network are you running telliot on?
- what's your python version?

**Comment by tallywiesenberg** [comment]
I suspect this could be that the request to gemini failed, but the failure was not handled. then, `pydantic` tried to create a new `GeminiPriceResponse` from the request repsonse data (of which none was returned)

TLDR: Telliot didn't realize an API request failed and freaked out

@Spuddyminer did Telliot quit out here?

**Comment by Spuddyminer** [comment]
Telliot did quit here. Command is
`telliot-examples -a myaccountname report`
and I think the error happens when the query type rotation gets to TRB. 
Network is `rinkeby`
python version is 3.9.12
